src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: FC<Props> = ({ todos }) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
```

Comment:
return statement is not necessary here, we can return right away by wrapping component result in the round braces instead of the curly ones